### ISSUE TITLE
Support groupings of origin wikis in data (many-to-one)

### ISSUE THREAD

**Issue by KevinPayravi** [KevinPayravi/indie-wiki-buddy#115]
Currently, our data objects have a single origin (non-indie) wiki mapping to a destination (indie) wiki, as follows:

```
{
  "id": "en-example",
  "origin": "Example Fandom Wiki",
  "origin_base_url": "example.fandom.com",
  "origin_content_path": "/wiki/",
  "destination": "Example Wiki",
  "destination_base_url": "example.com",
  "destination_content_path": "/w/",
  "destination_platform": "mediawiki",
  "destination_icon": "example.png"
}
```

There are cases where it would be useful to be able to map multiple origin wikis to a destination wiki (see https://github.com/KevinPayravi/indie-wiki-buddy/issues/101 for an example; there are 7 Friends at the Table wikis on Fandom that consolidated into one independent wiki). Other potential use cases would be when a topic has non-independent wikis across multiple platforms (e.g. both Fandom and Fextralife). By grouping these together under a single toggle, we would save a good amount of space on the UI + make it more convenient to toggle all wikis on a specific topic.

A new data structure could have the origin wikis in their own array, like:
```
{
  "id": "en-example",
  "label": "Example Fandom Wikis",
  "origins": [
    {
      "origin": "Example Fandom Wiki",
      "origin_base_url": "example.fandom.com",
      "origin_content_path": "/wiki/"
    },
    {
      "origin": "Example Fandom Wiki 2",
      "origin_base_url": "example2.fandom.com",
      "origin_content_path": "/wiki/"
    }
  ],
  "destination": "Example Wiki",
  "destination_base_url": "example.com",
  "destination_content_path": "/w/",
  "destination_platform": "mediawiki",
  "destination_icon": "example.png"
}
```

**Comment by KevinPayravi** [comment]
Work has been completed on this, will be in the next release.

**Comment by KevinPayravi** [comment]
Now available in version `2.0.0`
